x-pack/plugins/security_solution/public/management/components/console/console.tsx

```diff
@@ -39,8 +39,7 @@ const ConsoleWindow = styled.div`
     }
 
     &-container {
-      padding: ${({ theme: { eui } }) => eui.euiSizeL} ${({ theme: { eui } }) => eui.euiSizeL}
-        ${({ theme: { eui } }) => eui.euiSizeL} ${({ theme: { eui } }) => eui.euiSizeL};
+      padding: ${({ theme: { eui } }) => eui.euiPanelPaddingModifiers.paddingMedium};
```

Comment:
@paul-tavares I think you have modified this too. Following the mocks, the padding seems `16px` right?